examples/AffinePlaneCurve.jl

```diff
@@ -224,11 +220,7 @@ function Oscar.isreduced(C::AffinePlaneCurve)
      D = factor(C.eq)
      C.components = Dict(AffinePlaneCurve(x) => D.fac[x] for x in keys(D.fac))
   end
-  if [C.components[x] for x in keys(C.components)] == [1 for x in keys(C.components)]
-     return true
-  else
-     return false
-  end
+  return all(x -> x == 1, values(C.components))
```

Comment:
As above, perhaps like this?
```suggestion
  return all(isone, values(C.components))
```